package.json

```
@@ -0,0 +1,57 @@
+{
+    "name": "@okp4/subql-okp4",
+    "version": "0.1.0",
+    "private": false,
+    "description": "SubQuery Project for the OKP4 protocol",
+    "main": "dist/index.js",
+    "scripts": {
+        "build": "yarn subql build",
+        "codegen": "yarn subql codegen",
+        "start:docker": "docker-compose pull && docker-compose up --remove-orphans",
+        "dev": "yarn subql codegen && yarn subql build && docker-compose pull && docker-compose up --remove-orphans",
+        "prepack": "rm -rf dist && npm run build",
```

Lingering npm call here 👇 
```suggestion
        "prepack": "rm -rf dist && yarn build",
```